src/main/webapp/i18n/en/gradingSystem.json

```diff
@@ -45,8 +51,10 @@
             },
             "error": {
                 "empty": "Grade steps are empty!",
+                "negativeMaxPoints": "Maximum points can't be negative",
                 "emptyFields": "All grade step fields must be filled out!",
```

Comment:
Not in the scope of this PR but still relevant for the next time you work on the grade steps: this should be changed to either "...must be filled" or "...must be filled in" (AFAIK filled out is not correct)

Reply:
Could be. I know there's the "fill out a form" phrase in English.  Haven't really looked that up explicitly when I wrote this.